### PR TITLE
feat: cs-1905 Add tab to manage Variable Assignments when editing PDF

### PR DESCRIPTION
**Pull request by athena-chen-chen**
https://user-images.githubusercontent.com/58053108/215902195-935461e5-0489-4117-ad05-6dd69b4f9249.mov

Screen Shot 2023-01-31 at 2 28 58 PM
Screen Recording 2023-01-31 at 3.47.36 PM

### CODE REVIEW

File: apps/tenant-management-webapp/src/app/pages/admin/services/pdf/templates/previewEditor/TemplateEditor.tsx [hunk 1]
```diff
@@ -218,6 +223,24 @@ export const TemplateEditor: FunctionComponent<TemplateEditorProps> = ({
                 </GoAFormItem>
               </>
             </Tab>
+            <Tab
+              testId={`pdf-test-generator`}
+              label={<PdfEditorLabelWrapper>Template Variables</PdfEditorLabelWrapper>}
```
Comment (smettu007) [inline apps/tenant-management-webapp/src/app/pages/admin/services/pdf/templates/previewEditor/TemplateEditor.tsx:228]: it should be `Variable Assignments` i think as per the story https://goa-dio.atlassian.net/jira/software/projects/CS/boards/13?selectedIssue=CS-1904

File: apps/tenant-management-webapp/src/app/pages/admin/services/pdf/testGenerate/generatePDF.tsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+          }}
+        >
+          Reset PDF
+        </GoAButton>
+      </FlexRow>
+    </>
+  );
+};
+
+const EditorStyles = styled.div`
```
Comment (smettu007) [inline apps/tenant-management-webapp/src/app/pages/admin/services/pdf/testGenerate/generatePDF.tsx:94]: please move these to its style-components file

File: apps/tenant-management-webapp/src/app/pages/admin/services/pdf/templates/previewEditor/TemplateEditor.tsx (hunk 1 above)
Comment (athena-chen-chen) [reply apps/tenant-management-webapp/src/app/pages/admin/services/pdf/templates/previewEditor/TemplateEditor.tsx:228]: The design picture shows "Template Variables".

Comment (smettu007) [reply apps/tenant-management-webapp/src/app/pages/admin/services/pdf/templates/previewEditor/TemplateEditor.tsx:228]: image
there is a note that mentions it

File: apps/tenant-management-webapp/src/app/pages/admin/services/pdf/testGenerate/generatePDF.tsx (hunk 2 above)
Comment (athena-chen-chen) [reply apps/tenant-management-webapp/src/app/pages/admin/services/pdf/testGenerate/generatePDF.tsx]: Moved, Thanks

**Pull request merged**